pattoo_shared/installation/packages.py

```diff
@@ -2,60 +2,26 @@
 # Standard imports
 import os
 import getpass
-import json
-import urllib.request
 
 # Import pattoo related libraries
 from pattoo_shared.installation import shared
 from pattoo_shared import log
 
 
-def get_installed_packages():
-    """Retrieve installed pip packages.
+def get_package_version(package_name):
+    """Retrieve installed pip package version.
 
     Args:
-        None
+        package_name: The name of the package
 
     Returns:
         package_dict:A dictionary containing the installed packages
 
     """
-    packages = shared.run_script('python3 -m pip freeze')[1]
-    # Retrieve package names
-    keys = [package.decode().split('==')[0] for package in packages.split()]
-
-    # Retrieve package versions
-    values = [package.decode().split('==')[1] for package in packages.split()]
-
-    # Create dictionary with package versions
-    package_dict = dict(zip(keys, values)) 
-
-    return package_dict
-
-
-def version_check(package_name, package_dict):
-    """Check if package installed is updated to its latest version.
-
-    Args:
-        package_dict: A dictionary containing the packages
-        package_name: The package being checked
-
-    Returns:
-        True: If the package is updated
-        False: If the package is not updated/installed
-
-    """
-    package_url = 'https://pypi.org/pypi/{}/json'.format(package_name)
-    # Retrieve information on latest package version
-    package_data = urllib.request.urlopen(package_url).read()
-    data = json.loads(package_data)
-    latest_version = data['info']['version']
-
-    # Get version name from dictionary
-    if package_dict.get(package_name) == latest_version:
-        return True
-    else:
-        return False
+    raw_description = shared.run_script('pip3 show {}'.format(package_name))[1]
+    pkg_description = raw_description.decode().split('\n')
+    version = pkg_description[1].replace(' ', '').split(':')
+    return version[1]
 
 
 def install_missing_pip3(package, verbose=False):
```

Comment:
The `test_install_missing_pip3 `unittest does not verify the version of a package it successfully installs. ie, does the installed version match the desired version after installation?